[debug] [2021-01-01T13:16:34.556Z] ----------------------------------------------------------------------
[debug] [2021-01-01T13:16:34.558Z] Command:       W:\nodejs\node.exe C:\Users\pyomt\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js emulators:start --import=./firestore-data --only functions,firestore
[debug] [2021-01-01T13:16:34.559Z] CLI Version:   8.17.0
[debug] [2021-01-01T13:16:34.559Z] Platform:      win32
[debug] [2021-01-01T13:16:34.559Z] Node Version:  v12.16.1
[debug] [2021-01-01T13:16:34.560Z] Time:          Fri Jan 01 2021 22:16:34 GMT+0900 (GMT+09:00)
[debug] [2021-01-01T13:16:34.560Z] ----------------------------------------------------------------------
[debug] [2021-01-01T13:16:34.560Z] 
[debug] [2021-01-01T13:16:34.570Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2021-01-01T13:16:34.571Z] > authorizing via signed-in user
[info] i  emulators: Starting emulators: functions, firestore {"metadata":{"emulator":{"name":"hub"},"message":"Starting emulators: functions, firestore"}}
[debug] [2021-01-01T13:16:36.594Z] [hub] writing locator at C:\Users\pyomt\AppData\Local\Temp\hub-fukuuchierp.json
[info] i  firestore: Detected non-emulator Firestore export at C:\Users\pyomt\Desktop\FukuUchi_ERP\functions\firestore-data {"metadata":{"emulator":{"name":"firestore"},"message":"Detected non-emulator Firestore export at C:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\functions\\firestore-data"}}
[warn] !  functions: The following emulators are not running, calls to these services from the Functions emulator will affect production: auth, database, hosting, pubsub {"metadata":{"emulator":{"name":"functions"},"message":"The following emulators are not running, calls to these services from the Functions emulator will affect production: \u001b[1mauth, database, hosting, pubsub\u001b[22m"}}
[info] +  functions: Using node@12 from host. {"metadata":{"emulator":{"name":"functions"},"message":"Using node@12 from host."}}
[debug] [2021-01-01T13:16:38.633Z] defaultcredentials: writing to file C:\Users\pyomt\AppData\Roaming\firebase\satounbd_gmail_com_application_default_credentials.json
[debug] [2021-01-01T13:16:38.637Z] Setting GAC to C:\Users\pyomt\AppData\Roaming\firebase\satounbd_gmail_com_application_default_credentials.json {"metadata":{"emulator":{"name":"functions"},"message":"Setting GAC to C:\\Users\\pyomt\\AppData\\Roaming\\firebase\\satounbd_gmail_com_application_default_credentials.json"}}
[info] i  firestore: Importing data from C:\Users\pyomt\Desktop\FukuUchi_ERP\functions\firestore-data\firestore-data.overall_export_metadata {"metadata":{"emulator":{"name":"firestore"},"message":"Importing data from C:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\functions\\firestore-data\\firestore-data.overall_export_metadata"}}
[warn] !  firestore: Cloud Firestore rules file C:\Users\pyomt\Desktop\FukuUchi_ERP\firestore.rules specified in firebase.json does not exist. {"metadata":{"emulator":{"name":"firestore"},"message":"Cloud Firestore rules file \u001b[1mC:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\firestore.rules\u001b[22m specified in firebase.json does not exist."}}
[warn] !  firestore: The emulator will default to allowing all reads and writes. Learn more about this option: https://firebase.google.com/docs/emulator-suite/install_and_configure#security_rules_configuration. {"metadata":{"emulator":{"name":"firestore"},"message":"The emulator will default to allowing all reads and writes. Learn more about this option: https://firebase.google.com/docs/emulator-suite/install_and_configure#security_rules_configuration."}}
[debug] [2021-01-01T13:16:40.651Z] Ignoring unsupported arg: projectId {"metadata":{"emulator":{"name":"firestore"},"message":"Ignoring unsupported arg: projectId"}}
[debug] [2021-01-01T13:16:40.651Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"firestore"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2021-01-01T13:16:40.652Z] Starting Firestore Emulator with command {"binary":"java","args":["-Duser.language=en","-jar","C:\\Users\\pyomt\\.cache\\firebase\\emulators\\cloud-firestore-emulator-v1.11.9.jar","--host","localhost","--port",10080,"--seed_from_export","C:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\functions\\firestore-data\\firestore-data.overall_export_metadata","--functions_emulator","localhost:5001"],"optionalArgs":["port","webchannel_port","host","rules","functions_emulator","seed_from_export"],"joinArgs":false} {"metadata":{"emulator":{"name":"firestore"},"message":"Starting Firestore Emulator with command {\"binary\":\"java\",\"args\":[\"-Duser.language=en\",\"-jar\",\"C:\\\\Users\\\\pyomt\\\\.cache\\\\firebase\\\\emulators\\\\cloud-firestore-emulator-v1.11.9.jar\",\"--host\",\"localhost\",\"--port\",10080,\"--seed_from_export\",\"C:\\\\Users\\\\pyomt\\\\Desktop\\\\FukuUchi_ERP\\\\functions\\\\firestore-data\\\\firestore-data.overall_export_metadata\",\"--functions_emulator\",\"localhost:5001\"],\"optionalArgs\":[\"port\",\"webchannel_port\",\"host\",\"rules\",\"functions_emulator\",\"seed_from_export\"],\"joinArgs\":false}"}}
[info] i  firestore: Firestore Emulator logging to firestore-debug.log {"metadata":{"emulator":{"name":"firestore"},"message":"Firestore Emulator logging to \u001b[1mfirestore-debug.log\u001b[22m"}}
[debug] [2021-01-01T13:16:42.530Z] API endpoint: http://localhost {"metadata":{"emulator":{"name":"firestore"},"message":"API endpoint: http://localhost"}}
[debug] [2021-01-01T13:16:42.530Z] :10080
If you are using a library that supports the FIRESTORE_EMULATOR_HOST environment variable, run:

   export FIRESTORE_EMULATOR_HOST=localhost:10080

Dev App Server is now running.

 {"metadata":{"emulator":{"name":"firestore"},"message":":10080\nIf you are using a library that supports the FIRESTORE_EMULATOR_HOST environment variable, run:\r\n\n   export FIRESTORE_EMULATOR_HOST=localhost:10080\n\nDev App Server is now running.\n\r\n"}}
[debug] [2021-01-01T13:16:46.688Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"ui"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2021-01-01T13:16:46.688Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"ui"},"message":"Ignoring unsupported arg: port"}}
[debug] [2021-01-01T13:16:46.688Z] Starting Emulator UI with command {"binary":"node","args":["C:\\Users\\pyomt\\.cache\\firebase\\emulators\\ui-v1.3.0\\server.bundle.js"],"optionalArgs":[],"joinArgs":false} {"metadata":{"emulator":{"name":"ui"},"message":"Starting Emulator UI with command {\"binary\":\"node\",\"args\":[\"C:\\\\Users\\\\pyomt\\\\.cache\\\\firebase\\\\emulators\\\\ui-v1.3.0\\\\server.bundle.js\"],\"optionalArgs\":[],\"joinArgs\":false}"}}
[info] i  ui: Emulator UI logging to ui-debug.log {"metadata":{"emulator":{"name":"ui"},"message":"Emulator UI logging to \u001b[1mui-debug.log\u001b[22m"}}
[debug] [2021-01-01T13:16:46.810Z] Web / API server started at http://localhost:4000
 {"metadata":{"emulator":{"name":"ui"},"message":"Web / API server started at http://localhost:4000\n"}}
[info] i  functions: Watching "C:\Users\pyomt\Desktop\FukuUchi_ERP\functions" for Cloud Functions... {"metadata":{"emulator":{"name":"functions"},"message":"Watching \"C:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\functions\" for Cloud Functions..."}}
[debug] [2021-01-01T13:16:47.214Z] [worker-pool] addWorker(~diagnostic~) {"metadata":{"emulator":{"name":"functions"},"message":"[worker-pool] addWorker(~diagnostic~)"}}
[debug] [2021-01-01T13:16:47.214Z] [worker-pool] Adding worker with key ~diagnostic~, total=1 {"metadata":{"emulator":{"name":"functions"},"message":"[worker-pool] Adding worker with key ~diagnostic~, total=1"}}
[debug] [2021-01-01T13:16:47.214Z] [worker-pool] submitWork(triggerId=) {"metadata":{"emulator":{"name":"functions"},"message":"[worker-pool] submitWork(triggerId=)"}}
[debug] [2021-01-01T13:16:47.215Z] [worker-~diagnostic~-1b248510-19af-4b4f-84e4-d6e02341ec69]: Assigning socketPath: \\?\pipe\C:\Users\pyomt\Desktop\FukuUchi_ERP\functions\16876 {"metadata":{"emulator":{"name":"functions"},"message":"[worker-~diagnostic~-1b248510-19af-4b4f-84e4-d6e02341ec69]: Assigning socketPath: \\\\?\\pipe\\C:\\Users\\pyomt\\Desktop\\FukuUchi_ERP\\functions\\16876"}}
[debug] [2021-01-01T13:16:47.215Z] [worker-~diagnostic~-1b248510-19af-4b4f-84e4-d6e02341ec69]: BUSY {"metadata":{"emulator":{"name":"functions"},"message":"[worker-~diagnostic~-1b248510-19af-4b4f-84e4-d6e02341ec69]: BUSY"}}
